requirements.txt

```diff
@@ -61,3 +61,6 @@ urllib3==1.26.6; python_version >= "2.7" and python_full_version < "3.0.0" or py
 uvicorn==0.15.0 \
     --hash=sha256:17f898c64c71a2640514d4089da2689e5db1ce5d4086c2d53699bf99513421c1 \
     --hash=sha256:d9a3c0dd1ca86728d3e235182683b4cf94cd53a867c288eaeca80ee781b2caff
+prefect==1.1.0 \
```

Comment:
Ah we're using Poetry for our dependency management as per the Readme, so you'll need to add prefect to the pyproject.toml. You can install poetry first, `pip install poetry`, then add prefect with `poetry add prefect`